channeld/channeld.c

```diff
@@ -3441,6 +3478,38 @@ static struct inflight *inflights_new(struct peer *peer)
 	return inf;
 }
 
+static void update_hsmd_with_splice(struct peer *peer, struct inflight *inflight,
+				    const enum tx_role our_role,
+				    const struct amount_msat push_val)
+{
+	u8 *msg;
+
+	/* local_upfront_shutdown_script, local_upfront_shutdown_wallet_index,
+	 * remote_upfront_shutdown_script aren't allowed to change, so we
+	 * don't need to gather them */
+	msg = towire_hsmd_ready_channel(
+		NULL,
+		peer->channel->opener == LOCAL,
+		inflight->amnt,
```

Comment:
is this the post-splice channel value for this candidate?

Reply:
yes it is calculated from this call https://github.com/vincenzopalazzo/lightning/blob/master/channeld/channeld.c#L3546

So yes I think this is the total amount after one of the splice candidate is locked